src/components/cards/MetadataCard.jsx

```diff
@@ -34,38 +64,57 @@ export default function MetadataCard({
       }
     >
       <List dense>
-        {metadata.map(datum => {
-          const value = datum.value || datum.defaultValue;
-          if (!value) return null;
-          return (
-            <ListItem key={datum.id}>
-              {datum.icon && (
-                <ListItemIcon style={{ minWidth: 36 }}>
-                  <datum.icon />
-                </ListItemIcon>
-              )}
-              <ListItemText
-                primary={
-                  <Text
-                    component="span"
-                    variant="caption"
-                    id={datum.titleId}
+        {metadata
+          ? metadataToDisplay.map(field => {
+              const viewComponentProps =
+                field.viewComponentProps || {};
```

Comment:
Should all of these field. accessors be field?.  ?

Reply:
I guess not since we know field exists. I'm just (irrationally?) worried that one of these will not work.

Reply:
yeah fair enough. couldn't hurt